src/compositions/Accordion/Accordion.tsx

```diff
@@ -72,8 +72,7 @@ const AriaAccordionItem: FC<AriaAccordionItemProps> = ({ item, state, header })
                 >
                     <div
                         {...regionProps}
-                        className="px-8 pb-7 mt--1"
-                        style={{ visibility: isOpen ? "visible" : "hidden" }}
+                        className={`tw-px-8 tw-pb-7 tw-mt-1 ${isOpen ? "tw-visible" : "tw-invisible"}`}
```

Comment:
> Tailwind is smart enough to generate classes like -mx-72 when it sees the leading dash, not mx--72 like you might expect.

Reply:
![image](https://user-images.githubusercontent.com/2420769/128517668-c2137121-6ded-43ec-80c2-7b60e88a9e97.png)
actually it's not a typo 😉

Reply:
i use to not need to change the y-padding on the accordion header, because this lead to a jumping effect with framer-motion 🙈

Reply:
why i needed to this, compare the whole header/accordion with the figma design...